CONTRIBUTING.md

```diff
@@ -524,6 +490,39 @@ into any of those categories, use your best judgement.
 * Use `HasX` for protocols that have readable and/or writable attributes
   or getter/setter methods (e.g. `HasItems`, `HasFileno`).
 
+### Incomplete stubs
+
+We accept partially annotated stubs, especially for larger packages.
+These need to follow the following guidelines:
+
+* A package must include all public submodules, all public classes,
+  functions and methods, and all other public fields.
+* Included functions and methods must list all arguments, but the argument
+  types and the return type can be left unannotated or only be partially
+  annotated.
+* A type can be partially annotated by using the `_typeshed.Incomplete`
```

Comment:
Why is,

```python
def foo(x) -> int:
  ...
```

preferred over,

```python
def foo(x: Incomplete) -> int:
  ...
```

The latter I can grep for easily to eventually fix it. The former feels easier to forget.

Reply:
See #8472 for where we discussed this point in depth

Reply:
> The latter I can grep for easily to eventually fix it. 

Alternatively you can check the stub under pyright stricter config and they'll all show up.